samples/drivers/led_is31fl3216a/src/main.c

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2023 Endor AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/led.h>
+#include <zephyr/kernel.h>
+
+#define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
```

Comment:
Remove that and use `LOG_MODULE_REGISTER(main, CONFIG_LOG_DEFAULT_LEVEL);`

Reply:
:+1: